crypto/pkcs7/pkcs7_test.cc

```diff
@@ -1526,7 +1526,6 @@ TEST(PKCS7Test, GettersSetters) {
   X509_ALGOR *md_alg2 = X509_ALGOR_new();
   X509_ALGOR_set_md(md_alg2, EVP_sha256());
   sk_X509_ALGOR_push(p7->d.sign->md_algs, md_alg2);
-
   p7si = PKCS7_SIGNER_INFO_new();
   ASSERT_TRUE(p7si);
```

Comment:
NP: Since deleting a blank like is the only change to this file, could we remove it from the commit?

Reply:
I accidentally inserted this in the previous PR when modifying this test, and didn't think removing this line merited its own PR.

Reply:
sure.